packages/ui/components/combobox/src/LionCombobox.js

```diff
@@ -141,6 +146,55 @@ export class LionCombobox extends LocalizeMixin(OverlayMixin(LionListbox)) {
     ];
   }
 
+  /**
+   * @override ChoiceGroupMixin
+   */
+  // @ts-ignore
+  get modelValue() {
+    const choiceGroupModelValue = super.modelValue;
+    if (choiceGroupModelValue !== '') {
+      return choiceGroupModelValue;
+    }
+    // Since the FormatMixin can't be applied to a [FormGroup](https://github.com/ing-bank/lion/blob/master/packages/ui/components/form-core/src/form-group/FormGroupMixin.js)
+    // atm, we treat it in a way analogue to InteractionStateMixin (basically same apis, w/o Mixin applied).
+    // Hence, modelValue has no reactivity by default and we need to call parser manually here...
+    return this.parser(this.value);
+  }
+
+  // Duplicating from ChoiceGroupMixin, because you cannot independently inherit/override getter + setter.
+  // If you override one, gotta override the other, they go in pairs.
+  /**
+   * @override ChoiceGroupMixin
+   */
+  set modelValue(value) {
+    super.modelValue = value;
+  }
+
+  /**
```

Comment:
```suggestion
  /**
   * We define the value getter/setter below as also defined in FormatMixin (since FormatMixin is meant for Formgroups/ChoiceGroup and Combobox is somewhat of an hybrid between a ChoiceGroup and Field, we copy them over to align with the LionField interface). 
   * 
```